src/channel.rs

```diff
@@ -1,25 +1,27 @@
 //! ADC input channels
 use crate::{ic, Ads1x1x, BitFlags as BF, Config};
 
-mod private {
-    pub trait Sealed {}
-}
+use private::ChannelSelection;
 
 /// Marker type for an ADC input channel.
-pub trait ChannelId<T>: private::Sealed {
+pub trait ChannelId<T> {
```

Comment:
Why should this trait not be sealed?

Reply:
It is actually sealed since the `ChannelSelection` type is private.